doc/changelog/07-vernac-commands-and-options/14336-deprecate-regular-subst-option.rst

```diff
@@ -0,0 +1,4 @@
+- **Deprecated:**
+  the Regular Subst Tactic option
```

Comment:
More precisely, this is a flag. And until it's removed, you can link to its documentation using `:flag:`.

Reply:
Hum, linking it means that when we remove it we will have to fix the changes file. Is that reasonable? I know we've been doing that already but it seems quite ugly to me.

Reply:
(Done anyways.)